<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithmeticOpDialogForm</class>
 <widget class="QDialog" name="ArithmeticOpDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>395</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arithmetic Operations</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Layer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_inputLayerLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="m_searchLayerToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Band:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="m_bandComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="m_addLayerPushButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../share/terralib/icons/terralib/scalable/media-playback-start-green.svg</normaloff>../../../../../../share/terralib/icons/terralib/scalable/media-playback-start-green.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextBrowser" name="m_operationTextEdit"/>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QPushButton" name="m_sqrtPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>sqrt</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="m_parRightPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>(</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="m_parLeftPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="m_sinPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>sin</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="m_asinPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>asin</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="m_expPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>^</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="m_cosPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>cos</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="m_acosPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>acos</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="m_logPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>log</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="m_tanPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>tan</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="m_atanPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>atan</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QPushButton" name="m_lnPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ln</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QPushButton" name="m_n7PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>7</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="m_n8PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="m_n9PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>9</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="m_n4PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>4</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="m_n5PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>5</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="m_n6PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>6</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="m_n1PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="m_n2PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="m_n3PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>3</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="m_n0PushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="m_dotPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>.</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QPushButton" name="m_signalPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+ -</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QPushButton" name="m_divPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>/</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="m_cePushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>CE</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="m_multPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>*</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="m_cPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>C</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="m_subPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="m_loadPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Load operation</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="m_addPushButton">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="m_savePushButton">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Save operation</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="m_normalizeCheckBox">
         <property name="text">
          <string>Normalize</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QWidget" name="m_outputWidget" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="m_helpPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>136</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_okPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_cancelPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ArithmeticOpDialogForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
